README.md

```diff
@@ -73,6 +73,7 @@ Variable | Description | Examples/Values | Default
  `MAESTRO_SKIP_BUILD` | Skips build and process following steps if true, leave empty otherwise | `true` 
  `NEW_RELIC_API_KEY` | A user API key from NewRelic for registering deploys | `NRAK-XXXXXXXXXXXXXXXXXXXXXXXXX`
  `NEW_RELIC_APP_ID` | NewRelic APM's application identifier | `123456789` 
+ `DEPLOY_WEBHOOK_URL` | A custom webhook URL for registering deploys | `https://webhook.url?apikey=api_value&cluster_id={{CLUSTER}}&service={{SERVICE}}&repository={{REPOSITORY}}`
```

Comment:
Here can be?
DEPLOY_WEBHOOK_URL=https://hook.us1.make.com/dt8elcq7atff1851avee324lel05vcgm?apikey=a1b2c3d4
and on deploy.sh concatenate with the rest of the code, like this:
deploy_webhook_full_url="$DEPLOY_WEBHOOK_URL&cluster=$deploy_cluster_id&service=$deploy_service_name&repository=$deploy_repository_slug"

Reply:
@fagianijunior a custom webhook will have its own parameters and implementation, it is up to the third party to use as it suits or even ignore the proposed values. We could also make another values available if we find them important to enrich the notification... in a future version we also could get it even more complete by allowing a `POST` and custom headers sent along. I'd keep it as is for now unless you've got a better idea.